pkgs/development/libraries/wolfssl/default.nix

```
@@ -7,19 +7,21 @@
 
 stdenv.mkDerivation rec {
   pname = "wolfssl";
-  version = "5.4.0";
+  version = "5.5.0";
 
   src = fetchFromGitHub {
     owner = "wolfSSL";
     repo = "wolfssl";
     rev = "v${version}-stable";
-    sha256 = "sha256-5a83Mi+S+mASdZ6O2+0I+qulsF6yNUe80a3qZvWmXHw=";
+    sha256 = "sha256-PqxwWPK5eBcS5d6e0CL4uZHybDye1K8pxniKU99YSAE=";
   };
 
   postPatch = ''
     patchShebangs ./scripts
     # ocsp tests require network access
     sed -i -e '/ocsp\.test/d' -e '/ocsp-stapling\.test/d' scripts/include.am
+    substituteInPlace scripts/ocsp-stapling2.test \
```

```suggestion
    # ensure test detects musl-based systems too
    substituteInPlace scripts/ocsp-stapling2.test \
```

Else someone will come along and remove it thinking it makes no difference.

---

:+1: